mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -53,6 +53,22 @@ def _check_h5(fname):
         except Exception:
             raise SkipTest('h5py module required')
 
+@requires_h5py
+@testing.requires_testing_data
+@pytest.mark.parametrize('fnames', [raw_mat_fnames, raw_h5_fnames])
+def test_read_events_eeglab_warnings(fnames, tmpdir):
+    """Test importing EEGLAB .set files."""
+    tmpdir = str(tmpdir)
+    raw_fname, raw_fname_onefile = fnames
+    with pytest.warns(RuntimeWarning) as w:
+        read_raw_eeglab(input_fname=raw_fname, preload=True, montage=montage)
+        read_raw_eeglab(input_fname=raw_fname_onefile, preload=True, montage=montage)
+        read_raw_eeglab(input_fname=raw_fname, preload=False, montage=montage)
+        read_raw_eeglab(input_fname=raw_fname_onefile, preload=False, montage=montage)
+    for want in ('Events like', 'consist entirely', 'could not be mapped',
+                 'string preload is not supported'):
+        assert (any(want in str(ww.message) for ww in w))
```

Comment:
You should probably loop over params, otherwise there is no guaranteed each `read_raw_eeglab` emits all of these, only a guarantee that at least one of the `read_raw_eeglab` calls emits one of each